l10n_fi_edicode/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    edicode = fields.Char(string="Edicode")
+    einvoice_operator_id = fields.Many2one(
+        comodel_name="res.partner.operator.einvoice",
+        string="eInvoice Operator",
+        old_name="einvoice_operator",
```

Comment:
Support for `old_name` was dropped in Odoo 13. Please create a migration script instead under the `migrations` folder.

Reply:
done